lib/util/auth.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  isAccessTokenExpired(req) {
+    const { auth = {} } = req.session
+    const { expiryDate = null } = auth
+    if (expiryDate === null) {
+      return false
```

Comment:
❓  is `false` okay if `expiryDate` is null ?

Reply:
memo:

When the password auth and the third-party auth enabled at the same time and third party auth expired, should it jump to login page even if there is remaining password auth session?